src/query/plan/operator.cpp

```diff
@@ -33,6 +31,7 @@
 #include "query/frontend/semantic/symbol_table.hpp"
 #include "query/interpret/eval.hpp"
 #include "query/path.hpp"
+#include "query/plan/operator.hpp"
```

Comment:
Actually this header was in the right spot, because this is the main header for the source file. That always goes first. Search "Includes" in our [public](https://memgraph.notion.site/C-Code-Conventions-7497766e0e1647b987733b9e94171423) code style guide.